packages/synthetix-main/contracts/interfaces/IAccountModule.sol

```diff
@@ -3,12 +3,20 @@ pragma solidity ^0.8.0;
 
 import "@synthetixio/core-contracts/contracts/interfaces/ISatelliteFactory.sol";
 import "@synthetixio/core-modules/contracts/interfaces/INftModule.sol";
+import "@synthetixio/core-contracts/contracts/utils/SetUtil.sol";
```

Comment:
Wouldn't it be more appropriate to import SetUtil in AccountModule?

Reply:
removed